libs/covid/data-api/src/lib/modules/counties/counties.service.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository, Like } from 'typeorm';
 
-import { County, CountyClaim, StatusType, EntityStatus, TestingSite } from '@tamu-gisc/covid/common/entities';
+import { County, CountyClaim, StatusType, EntityStatus, TestingSite, Lockdown } from '@tamu-gisc/covid/common/entities';
```

Comment:
Cleanup unused imports

Reply:
Lockdown  is used in the ClaimWithData interface

Reply:
`StatusType` is the odd one left out. That was my doing I think.